docs/UserGuide.md

```diff
@@ -284,7 +286,18 @@ Format: `focus INDEX`
 * To switch to another Candidate's information, user will just need to type the new command and the new index of the candidate.
 
 ## Scheduling interviews
-
+**:information_source: Note:** 
+
+* The duration of an interview is fixed at 30 minutes. A new feature to allow the user to schedule interviews of flexible duration is
+under consideration for future versions of TAlent Assistant™.
+* Interviews that have expired will automatically be deleted from the interview schedule. An interview is considered
```

Comment:
Perhaps we can clarify that they will be automatically deleted `upon restarting the application` - if im correct this is the expected behaviour :O